libfive/include/libfive/render/brep/mesh.hpp

```diff
@@ -21,6 +29,36 @@ struct BRepSettings;
 template <unsigned N> class Region;
 
 class Mesh : public BRep<3> {
+  /////////////////////////////////////////////////////////
+  // |-|4><0r3D UP, d00d, all your base are belong to us //
+  /////////////////////////////////////////////////////////
+public:
+  static std::stringstream h4x0r3dN0rm5; // ^|||^
+  static std::stringstream h4x0r3dV3r75; // \. ./ sceming ponee wuz here
```

Comment:
wat

Reply:
Joking, I was pretending to be the great Lisp hacker "Scheming Pony" get the temporary modifications I'm doing with OBJ format into master.  I will refrain from trying to be humorous ever again.